packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 3.0.0
+- Secure Credentials Storage for SuiteCloud SDK
+- Removal of OAuth 1.0 Token-Based Authentication support in SuiteCloud SDK
```

Comment:
This is not correct, lets remove the second line.
From past release we only work with OAuht2.0 in vscode-extension and in suitecloud-sdk tools in general.